infra/base-images/base-builder/install_go_CodeIntelligenceTesting.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -eux
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Install CodeIntelligenceTesting Go
+#  (require at least Go1.4 to boostrap)
+cd "/root"
+git clone https://github.com/CodeIntelligenceTesting/go.git .go-CodeIntelligenceTesting
+cd ".go-CodeIntelligenceTesting/src"
+git checkout dev.libfuzzer.18
+# Disable tests, which fails
+sed -i '/^exec .* tool dist test -rebuild "$@"/ s/./#&/' run.bash
+./all.bash
+
+# Remember to set env in dockerfile before this line
```

Comment:
Is this comment necessary? It's not mentioned in the original `install_go.sh` either so we can skip this.

Reply:
I probably should remove [the next two lines](https://github.com/google/oss-fuzz/blob/8e590f77d1a84084d6dc3186521ae8b2db20c3ba/infra/base-images/base-builder/install_go_CodeIntelligenceTesting.sh#L29-L30) as well: 
It seems to me that they only print out how to `set` the environment variables (`GOCODEINTELLIGENCETESTINGPATH` & `PATH`) without making any actual effect.
I believe the (only) way to set environment variables is [using command `ENV` in the corresponding `Dockerfile`](https://github.com/google/oss-fuzz/blob/b1bc21aadb3a234ca43dd8bd5e7e4283c0c41f98/infra/base-images/base-builder-go-codeintelligencetesting/Dockerfile#L21).

Similar with [the original `Dockerfile`](https://github.com/google/oss-fuzz/blob/b1bc21aadb3a234ca43dd8bd5e7e4283c0c41f98/infra/base-images/base-builder-go/Dockerfile#L20-L24) and [the original `install_go.sh`](https://github.com/google/oss-fuzz/blob/a4cd74a98efe1266a4bdda079770baeb7fb2059a/infra/base-images/base-builder-new/install_go.sh#L24-L25).

I hope I did not misinterpret the code?

Reply:
Remove the comment and the two `echo`s in commit [Remove redundant lines](https://github.com/google/oss-fuzz/commit/d1ee10b0b918382e7cf830b78a674b9e6cbfeb8b).

Reply:
Yes fine to remove. The are printed them for debugging purposes.